mapdata/WarcraftLegacies/AbilityData/Skin/859058497.json

```
@@ -0,0 +1,41 @@
+{
+  "OldId": 1852130113,
+  "NewId": 859058497,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Id": 1718840929,
+      "Type": 3,
+      "Value": "(Wanderer)"
+    },
+    {
+      "Id": 2037082209,
+      "Type": 3,
+      "Value": "Q"
+    },
+    {
+      "Level": 1,
+      "Id": 828536161,
+      "Type": 3,
+      "Value": "Applies Ensnare to a target unit or hero. Air units that are ensnared can be attacked as though they were land units. |nLasts 10(2) seconds."
```

You still need to indicate what Ensnare does, on top of calling out that buff name.
```suggestion
      "Value": "Applies Ensnare to a target unit or hero, preventing it from moving. Air units that are ensnared can be attacked as though they were land units. |nLasts 10(2) seconds."
```